frontend/src/components/ConsultantRow.tsx

```diff
@@ -518,20 +521,48 @@ function DetailedBookingCell({
 
   return (
     <td className="h-8 p-0.5">
-      <input
-        type="number"
-        min="0"
-        step="7.5"
-        value={hours}
-        disabled={detailedBooking.bookingDetails.type == BookingType.Vacation}
-        onChange={(e) => setHours(Number(e.target.value))}
-        onFocus={() => setIsDisabledHotkeys(true)}
-        onBlur={() => updateHours()}
-        className={`small-medium rounded text-right w-full py-2 pr-2
-     ${getColorByStaffingType(
-       detailedBooking.bookingDetails.type ?? BookingType.Offer,
-     )} ${hours == 0 && "bg-opacity-30"}`}
-      ></input>
+      <div
+        className={`flex flex-row justify-center items-center rounded px-3 py-2 ${getColorByStaffingType(
+          detailedBooking.bookingDetails.type ?? BookingType.Offer,
+        )} ${hours == 0 && "bg-opacity-30"}`}
+      >
+        {isChangingHours && numWeeks <= 8 && (
+          <button
+            className="p-2 rounded-full hover:bg-primary/10 hidden lg:flex"
+            onClick={() => {
+              setHours(hours - 7.5), updateHours(-7.5);
+            }}
+          >
+            <Minus className="w-4 h-4" />
+          </button>
+        )}
+
+        <input
+          type="number"
+          min="0"
+          step="7.5"
+          value={hours}
+          disabled={detailedBooking.bookingDetails.type == BookingType.Vacation}
+          onChange={(e) => setHours(Number(e.target.value))}
+          onFocus={() => {
```

Comment:
Out of curiosity: how does this `() => { function1(a), function(b) }` notation work here?

Reply:
Det skal heller være ; her 😇